.github/workflows/openai_workflow.yml

```
@@ -0,0 +1,54 @@
+          OPENAI_API_KEY: ${{ secrets.OPENAI_API_KEY }}
+
+      - name: Queue Task
+        if: ${{ steps.quota.outputs.usage >= steps.quota.outputs.quota }}
+        run: |
+          task='{ "subscription": "OpenAI_API", "details": "..." }'
+          python scripts/quota_manager.py add_task "$task"
+          echo "OpenAI API quota reached. Task queued."
+
+      - name: Commit state changes
+        if: always()
+        run: ./scripts/commit_changes.sh "chore: update state for OpenAI task"
```

actionlint (1.7.8)

could not parse as YAML: mapping values are not allowed in this context

(syntax-check)